### DISCUSSION TITLE
glTF plug-in for UE5 ?

### DISCUSSION THREAD

**Discussion by motorsep** [code4game/glTFForUE4#55] [General]
Are there any plans to update this wonderful plug-in to UE5.0.1+ ?

**Comment by jixingcn** [comment]
Yes, in progress.

**Reply by RaiaN** [reply]
UE5 support is very much needed! :)

**Reply by jixingcn** [reply]
Sorry, I don't have time to do this quickly. You can use it in UE4 and then import the `.uasset` to UE5 if you can't wait.

**Comment by jixingcn** [comment]
@motorsep @RaiaN Now supports UE5 in the version 2.0.10. You can download the compiled package from itch.io.
